src/Types.ts

```diff
@@ -1000,6 +1000,15 @@ export type ModalizerElementAccessibleCheck = (
     activeModalizerElements?: HTMLElement[]
 ) => boolean;
 
+export interface UncontrolledProps {
+    // Indicated that uncontrolled area traps focus, that makes Tabster
+    // to give it full control over the Tab handling. If not set, Tabster
+    // might still handle Tab in the uncontrolled area, when, for example,
+    // there is an inactive Modalizer (that needs to be skipped) after the
+    // last focusable element of the uncontrolled area.
+    trapsFocus?: boolean;
```

Comment:
If this is to let tabster have full control of Tab handling why call it `trapFocus`? maybe something like `complete` or smth like that to indicate it's completely uncontrolled